app/src/main/java/com/pvp/app/ui/screen/calendar/TaskFormsCreate.kt

```diff
@@ -96,8 +97,6 @@ fun TaskCreateSheet(
 
             Spacer(modifier = Modifier.size(16.dp))
 
-            HorizontalDivider()
-
             Spacer(modifier = Modifier.size(8.dp))
```

Comment:
Spacers could be merged together into one and become 24.dp size or... Just the 8.dp spacer could be dropped, since we wanted a gap from the divider, which is not the case now.